src/state/error-handlers.js

```
@@ -21,12 +21,20 @@ function handleCustomError(notify, error) {
   return Promise.reject(error);
 }
 
+function handleImageUploadError(notify, error) {
+  console.error(error);
```

what do you think about using captureException instead of console.erroring?

---

ah i see your comment "We don't let errors bubble up to sentry if they're just file upload errors - more than likely it's the user's problem, not ours" if we know there are specific user errors here can we give the user more helpful feedback? are they uploading files that aren't images?

---

We don't know what the error was more specifically as far as I can tell - maybe the upload failed because the asset file was corrupted, it was the wrong type, or too big. 

The reason for `console.error` is that a tech-savvy user might look there for more info on what went wrong; I'm open to taking that out too if we don't want it there.

---

yah your reasoning makes sense to me!